packages/component/src/Activity/ActivityFeedback.tsx

```diff
@@ -40,14 +73,61 @@ function ActivityFeedback({ activity }: ActivityFeedbackProps) {
     return Object.freeze(new Set([] as OrgSchemaAction[]));
   }, [graph, messageThing?.potentialAction]);
 
-  return (
-    <Feedback
-      actions={feedbackActions}
-      className={cx({
-        'webchat__thumb-button--large': feedbackActionsPlacement === 'activity-actions'
-      })}
-    />
+  const disclaimer = useMemo(
+    () =>
+      isFeedbackLoopSupported && isDefaultFeedbackActivity(activity)
+        ? activity.channelData.feedbackLoop?.disclaimer
+        : undefined,
+    [activity, isFeedbackLoopSupported]
+  );
+
+  const onFeedbackTypeChange = useCallback((newType?: string) => {
+    setFeedbackType(newType);
+    setShowFeedbackForm(newType !== undefined);
+    if (newType === undefined) {
```

Comment:
```suggestion
    if (typeof newType === 'undefined') {
```

Because, `window.undefined = 1` then your code will fail.